viewmaster/link.py

```
@@ -22,36 +37,74 @@
 
-# DEBUG_LOG_FILE = LOG_ROOT_PREFIX_ + 'link_debug.log'
-# debug_logfile = os.path.abspath(DEBUG_LOG_FILE)
-# debug_handler = pdslogger.file_handler(debug_logfile, level=logging.DEBUG,
-#                                        rotation='midnight')
-# LOGGER.add_handler(debug_handler)
+    Returns:
+        pdslogger.PdsLogger: Configured logger instance.
+    """
 
-################################################################################
+    LOGNAME = LOGNAME.replace('viewmaster', 'link')
```

Ruff (0.14.10)

`LOGNAME` may be undefined, or defined from star imports

(F405)